### DISCUSSION TITLE
Is there any way to have a global winbar like statusline?

### DISCUSSION THREAD

**Discussion by LinArcX** [neovim/neovim#24886] [Q&A] [answered]
from here: https://github.com/neovim/neovim/pull/17266, we can have a global statusline by setting: `laststatus = 3`. I wonder is there any same way for winbar?

**Comment by bfredl** [comment, marked as answer]
Isn't that just `'tabline'` ?

**Reply by LinArcX** [reply]
yes, tabline is what I wanted exactly. but.. why we still need winbar? is there any specific use case for that?
